CONTRIBUTING.md

```diff
@@ -329,6 +334,86 @@ If you already have the `refinebio-examples` docker image:
 docker run --mount type=bind,target=/home/rstudio,source=$PWD ccdl/refinebio-examples snakemake --cores 4
 ```
 
+## Pull Requests
+
+There are two protected branches in this repo:
+
+`staging` - where changes are initially sent for testing and previewing.
+`master` - where changes that were tested are made user-facing.
+
+This two branch set up allows us to make incremental changes that we can test before publishing to our user-facing github pages.
+It also means that pull requests have some extra methodology.
+
+#### Add to the testing branch: merges to staging
+
+After you've prepared the new or altered material for this repository, you can file a pull request to the `staging` branch (the default branch for this repository).
+The `staging` branch does not feed into the published, user-facing material so you can feel free to make your pull requests as iterative and incremental as it is useful.
+Use `squash and merge` when merging these pull requests into `staging` after approval.
+`squash and merge` will be helpful for us if we need to "cherry pick" commits if particular project or example is not yet ready for user-facing but other material is.
+
+After merging a pull request into `staging`, check that all the changes are correct by reviewing the newly generated pages using html preview.
+Go here and navigate to the pertinent pages that you've changed.
+`http://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/gh-pages-stages/01-getting-started/getting-started.html`
+You may also want to spot-check other pages to be sure that there have not been inadvertent or unexpected  changes introduced.
+
+#### Publish: staged changes merged to master
+
+Once a set of changes that are merged to `staging` are ready to be published, you can file a merge to `master` pull request.
+
+These types of PRs should only involve well-polished and ready for the public material.
+
+**Scenario 1: All changes from staging should be published**  
+
+- Create a new branch from the most up-to-date `master` branch, call it `publish-mychanges` where `mychanges` is a short tag relevant to the changes.
+- Checkout your newly created `publish` branch.
+- If you are certain that all changes in `staging` should be carried over to `master` (aka published to Github pages), then you can do a `git merge staging` into your new branch.
+- Now you can use the new branch to create the pull request to master as you would normally do.
```

Comment:
Once we merge `staging` into the new branch, will the next PR then automatically be created to merge to `master` instead of `staging`?